### PR TITLE
Constructionplanless build process

### PR DESCRIPTION
**Pull request by domtra**

### CODE REVIEW

File: gulpfile.js/tasks/copy.js [hunk 1]
```diff
@@ -3,11 +3,11 @@ const gulp = require('gulp')
...
     const changed = require('gulp-changed')
-    const handleErrors = require('../utils/handleErrors')
+    // const handleErrors = require('../utils/handleErrors')
     let task = gulp.src(config.copy)
-    .pipe(changed(config.dest))
-    .pipe(gulp.dest(config.dest))
-    .on('error', handleErrors)
+      .pipe(changed(config.dest))
+      .pipe(gulp.dest(config.dest))
+    // .on('error', handleErrors)
```
Comment (steffenbew) [inline gulpfile.js/tasks/copy.js]: remove?

File: gulpfile.js/tasks/stylus.js [hunk 2]
```diff
@@ -6,30 +6,30 @@ module.exports = function (config) {
     const autoprefixer = require('autoprefixer-stylus')
     const changed = require('gulp-changed')
     const gulpIf = require('gulp-if')
-    const handleErrors = require('../utils/handleErrors')
+    // const handleErrors = require('../utils/handleErrors')
```
Comment (steffenbew) [inline gulpfile.js/tasks/stylus.js]: remove?

File: gulpfile.js/tasks/copy.js (hunk 1 above)
Comment (domtra) [reply gulpfile.js/tasks/copy.js]: will be removed before merge in master. still doing some testing

File: gulpfile.js/tasks/stylus.js (hunk 2 above)
Comment (domtra) [reply gulpfile.js/tasks/stylus.js]: will be removed before merge in master. still doing some testing

**Pull request merged**